modal/sandbox.py

```diff
@@ -239,6 +239,13 @@ async def create(
         app_client: Optional[_Client] = None
 
         if app is not None:
+            if app.app_id is None:
+                raise ValueError(
+                    "App has not been initialized yet. To create an App lazily, use `App.lookup`: \n"
```

Comment:
Note I explicitly didn't mention `with app.run()` etc because it might confuse sandbox users more. They likely don't want to do that here.

Reply:
might want to also show a code sample of how to use it with a local app, idk

Reply:
Yeah I considered doing that, but I fear that will confuse users more. Can add it if you feel strongly about it.